cfpb/sbl-frontend
# [Task] Axios - Basic Implementation

Closes #202 

## Changes

- feat: Axios - Basic Implementation
- refactor: removed use of `fetch` and replaced with `axios` 

## How to test this PR

1. `yarn up -i axios`
2. `yarn start`
3. Make sure there are no regressions in the following fetches:

- fetchInstitutionDetails
- fetchAssociatedInstitutions
- fetchInstitutions
-  fetchIsDomainAllowed
-  fetchUserProfile
- submitUserProfile

## Future PRs
- Better Integration with `react-query`
- Intercept, Timeout, CancelToken, etc
- Error Handling with `react-query`
- Upload implementation
